packages/editable-html/src/editor.jsx

```diff
@@ -176,6 +176,8 @@ export class Editor extends React.Component {
   }
 
   componentDidMount() {
+    window.addEventListener('resize', this.onResize.bind(this));
```

Comment:
@andreeimiron - calling `fn.bind(...)` creates a new function - so we need to create the resize function once for this component and use it for in `addEventListener` and `removeEventListener`. 
```javascript
constructor(){
  this.onResize = (e) => {
    //...
  }
}
```